lib/rouge/lexers/opentype_feature_file.rb

```diff
@@ -66,6 +73,7 @@ def self.keywords
         end
 
         rule identifier, Name
+        rule %r/0x|\\[0-9A-Fa-f]+/, Num::Hex
```

Comment:
Should this be the following?

```suggestion
        rule %r/(?:0x|\\)[0-9A-Fa-f]+/, Num::Hex
```

Reply:
Yeah, that makes more sense.